src/commands/guild.ts

```diff
@@ -1,49 +1,59 @@
-import fetch from 'node-fetch';
-import { Message, MessageEmbed, MessageSelectMenu } from 'discord.js'
-import { userUtils } from '../utils/userUtils.js'
-import { responseHandler } from '../utils/responseHandler.js'
+import {Message, MessageEmbed} from 'discord.js'
+import axios from 'axios'
+import {responseHandler} from '../utils/responseHandler.js'
+import {userUtils} from '../utils/userUtils.js'
 
-export async function guild(message: Message, _arguments: string[]) {
-	const parsed = _arguments[0].replace(/[^0-9]/g,'')
-	const apiURL = `https://api.polytoria.com/v1/guild/info?id=${parsed}`;
+export async function guild(message: Message, args: string[]) {
+	const guildID = parseInt(args[0])
 
-	const response = await fetch(apiURL);
-	const data: any = await response.json();
-	const errresult = responseHandler.checkError(response,data)
-	if (errresult.HasError == true) return message.channel.send(`${errresult.DisplayText}`)
+	const response = await axios.get('https://api.polytoria.com/v1/guild/info', {params: {id: guildID}})
+	const data = response.data
 
-	if (data.Success !== true) return message.channel.send('There was an unexpected error.')
+	const errResult = responseHandler.checkError(response)
+
+	if (errResult.HasError == true) {
+		return message.channel.send(errResult.DisplayText)
+	}
 
 	const userData = await userUtils.getUserData(data.CreatorID)
 
 	const Embed = new MessageEmbed({
 		title: data.Name,
 		description: data.Description,
 		thumbnail: {
-			url: `${data.Thumbnail}`
+			url: data.thumbnail
```

Comment:
This does not work, in the response data thumbnail comes as `Thumbnail` and it's not considered a string for the Embed type for some reason.